data/mods/Aftershock/items/armor/exosuit/exosuit_spells.json

```diff
@@ -16,5 +16,152 @@
     "difficulty": -20,
     "max_level": 1,
     "energy_source": "NONE"
+  },
+  {
+    "type": "SPELL",
+    "id": "exo_flamethrower",
+    "name": "Exosuit Flamethrower",
+    "description": "A massive gout of flame leaps forward, igniting everything in it's path.",
+    "effect": "attack",
+    "shape": "cone",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": "IGNITE_FLAMMABLE",
+    "message": "",
+    "max_level": 20,
```

Comment:
Should this be 3 levels instead of twenty?  Or are damage and aoe intentionally maxed out on 3rd level?